src/utils/datetime_test.cljs

```diff
@@ -190,76 +193,75 @@
-          tomorrow?                 (fn [mocked curr-time]
-                                      (some #(= %
-                                                (-
-                                                 (int (get-month-day-int mocked))
-                                                 (int (get-month-day-int curr-time))))
-                                            [1 30 29 27]))
-          form-full-date            #(str (get-hh-mm %)
-                                          " " (string/capitalize (get-week-day %))
-                                          " " (get-month-day-int %)
-                                          " " (string/capitalize (get datetime/months (t/month %))))
-          today-date                #(str (get-hh-mm %) " today")
-          tomorrow-date             #(str (get-hh-mm %) " tomorrow")
-          write-date                #(cond (today? % curr-time)    (today-date %)
-                                           (tomorrow? % curr-time) (tomorrow-date %)
-                                           :else                   (form-full-date %))
-          will-unmute-in-1-hour     (remove-msecs (time-coerce/to-string mock-in-1-hour))
-          will-unmute-in-8-hours    (remove-msecs (time-coerce/to-string mock-in-8-hour))
-          will-unmute-in-15-mins    (remove-msecs (time-coerce/to-string mock-in-15-minutes))
-          will-unmute-in-two-days   (remove-msecs (time-coerce/to-string mock-in-two-days))
-          will-unmute-tomorrow      (remove-msecs (time-coerce/to-string mock-tomorrow))
-          will-unmute-in-three-days (remove-msecs (time-coerce/to-string mock-in-three-days))
-          will-unmute-in-four-days  (remove-msecs (time-coerce/to-string mock-in-four-days))
-          will-unmute-in-five-days  (remove-msecs (time-coerce/to-string mock-in-five-days))
-          will-unmute-in-six-days   (remove-msecs (time-coerce/to-string mock-in-six-days))]
-      (testing "Mute for minutes and hours"
-        (are [arg expected] (= (datetime/format-mute-till arg) expected)
-         will-unmute-in-15-mins (write-date mock-in-15-minutes)
-         will-unmute-in-1-hour  (write-date mock-in-1-hour)
-         will-unmute-in-8-hours (write-date mock-in-8-hour)))
-      (testing "Weekdays"
-        (are [arg expected] (= (datetime/format-mute-till arg) expected)
-         will-unmute-tomorrow      (write-date mock-tomorrow)
-         will-unmute-in-two-days   (write-date mock-in-two-days)
-         will-unmute-in-three-days (write-date mock-in-three-days)
-         will-unmute-in-four-days  (write-date mock-in-four-days)
-         will-unmute-in-five-days  (write-date mock-in-five-days)
-         will-unmute-in-six-days   (write-date mock-in-six-days)))
-      (testing "Until the user turns it back on"
-        (is (= "you turn it back on" (datetime/format-mute-till datetime/go-default-time))))))
+(deftest format-mute-till-test
```

Comment:
imo it seems like `format-mute-till` is quite application specific and this code is in a general `datetime` utilities. Is there a better location to have this code? 
cc @flexsurfer

Reply:
I think it'd be nicer if we have a muting_tests.cljs which has the muting specfic tests.
But for this case, It's a single test and it interacts with time functions so I think it's okish to have it here.

Reply:
I was referring to the functionality not to the test itself. It seems like this functionality is misplaced and ofc the test should sit alongside it 👍

Reply:
e.g src/contexts/common/muting seems more applicable imo

Reply:
Apologies for long response, Had this task on hold for a bit
Addressed here 351b87d9e335008028ad7382629f304646a4185e